condensedinlinepanel/templates/condensedinlinepanel/condensedinlinepanel.js

```diff
@@ -2,10 +2,14 @@
     CondensedInlinePanel.init('id_{{ self.formset.prefix }}', {
         summaryTextField: '{{ self.formset.summary_text_field|escapejs }}',
         canOrder: {{ self.formset.can_order|lower }},
+        panelLabel: "{{ self.label }}",
         renderCardHeader: function(form) {
             {% if self.card_header_from_field %}
                 {# Get the card header value from a field and escape it #}
                 var value = form.fields['{{ self.card_header_from_field|escapejs }}'];
+                if (value===null){
```

Comment:
What if the form is not new but the value of the specified field is ``null``? Perhaps we should check ``form.isNew`` here instead?